src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import classes from 'classnames';
+import logo from '../../assets/logo.svg';
+import logoSmall from '../../assets/logo_simple.svg';
+
+/**
+ * The Kids first logo.
+ * May be `full` with `Kids First Data Resource` titling or
+ * `small` with only the art asset.
+ */
+const Logo = ({title, kind, size, className}) => {
+
+  const logoAsset = kind === 'simple' ? logoSmall : logo;
+  const logoClasses = classes(
+      className,
+      'Logo',
+      {[`Logo--${size}`] : size !== 'medium'},
+  )
+  const titleClasses = classes(
+      'Logo--title',
+      {[`Logo--title-${size}`] : size !== 'medium'},
+  )
+  return (
+    <div className={logoClasses}>
+      <img
+        className={logoClasses}
+        alt='Kids First Data Resource Center logo'
+        src={logoAsset}
+      />
+      {(kind === 'simple' && title) &&
+        <h1 className={titleClasses}>{title}</h1>
+      }
+    </div>
+  )
+}
+
+Logo.propTypes = {
+  /** Title to be displayed when kind is `simple` */
+  title: propTypes.string,
+  /** The type of logo */
+  kind: propTypes.string,
+  /** Size of the logo */
+  size: propTypes.string,
+  /** Any additional classes to be applied to the logo */
+  className: propTypes.string,
+};
+
+Logo.defaultProps = {
+  title: null,
+  kind: 'full',
+  size: 'medium',
+  className: null,
+};
+
```

Comment:
Need to set the `displayName` so that the story source properly shows or figure out why the story source is truncating the component name